dnfdaemon-server/main.cpp

```diff
@@ -24,10 +24,25 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include <sdbus-c++/sdbus-c++.h>
 
 #include <iostream>
+#include <signal.h>
 
 static std::unique_ptr<SessionManager> session_manager = nullptr;
 
+void sigint_handler(int signum) {
+    std::cout << "received signal " << signum << std::endl;
```

Comment:
Is this output meant to stay? Should it print to `stderr` maybe? Even better, check `session_manager` is not null, pull the logger from it and write the message into the log? (does the daemon share the log with microdnf? interesting topic...)

Reply:
Oops, that's forgotten debug message. I'll remove it.